BE/src/main/java/todo3/codesquad/exception/TodoException.java

```diff
@@ -0,0 +1,4 @@
+package todo3.codesquad.exception;
+
+public class TodoException extends Exception{
```

Comment:
```suggestion
public class TodoException extends Exception {
```
사용하지 않는 custom exception 인가요?